packages/hooks/src/useGrowthbookFeatureFlag.ts

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react';
+import { Analytics } from '@deriv-com/analytics';
+
+interface UseGrowthbookFeatureFlagArgs<T> {
+    featureFlag: string;
+    defaultValue: T;
+}
+
+const useGrowthbookFeatureFlag = <T>({ featureFlag, defaultValue }: UseGrowthbookFeatureFlagArgs<T>) => {
+    const [featureFlagValue, setFeatureFlagValue] = useState<T>(defaultValue);
+
+    useEffect(() => {
+        const value = (Analytics?.getFeatureValue(featureFlag, defaultValue) || defaultValue) as T;
+        setFeatureFlagValue(value);
+
+        // Set the renderer for GrowthBook to update the value when the feature flag changes
+        Analytics.getInstances()?.ab?.GrowthBook?.setRenderer(() => {
```

Comment:
it's just for topic to discuss. Do you think that we have to add special function like that for analytics package?

Could you please describe what we getting with that func? 
I can't find a docs for that
Does it a real-time feature-flag switching?
It'll help me a lot :)